src/azure-cli/azure/cli/command_modules/serviceconnector/_help.py

```diff
@@ -217,13 +217,57 @@ def get_source_display_name(sourcename):
         # target resource params
         target_params = get_target_resource_params(target)
 
+        # special target resource to pass linter check with no auth params
+        if target == RESOURCE.ContainerApp:
+            helps['{source} connection create {target}'.format(source=source.value, target=target.value)] = """
+            type: command
+            short-summary: Create a {source_display_name} connection to {target}. No auth info is required.
```

Comment:
how about make it specific: {source_display_name} to aca. Or clearly discribe what does `no auth info is required` mean, the sentence is confusing